config/database.php

```
@@ -56,7 +56,7 @@
             'collation' => 'utf8mb4_unicode_ci',
             'prefix' => '',
             'prefix_indexes' => true,
-            'strict' => true,
+            'strict' => false,
```

I'm intrigued by this - what prompted you to turn strict mode off for the database? Not saying it should be always on, but wondering what your reasoning is. It's on by default to protect against data loss (i.e. returning an error if an INSERT statement includes invalid data, rather than just ignoring the error and inserting NULLs).

---

I forgot to turn this back to true as I was trying to troubleshoot something which had failed my foreign keys to get inserted via the migration, turning it to false was among the suggestions I found online but I later realized the problem and it had nothing to do with making it false. Thank you for the catch.